orekit/tests/validate_frames.py

```diff
@@ -0,0 +1,307 @@
+    Neptune_orekit,
+]
+OREKIT_FIXED_FRAMES = [body.getBodyOrientedFrame() for body in OREKIT_BODIES]
+
+# poliastro: bodies, inertial and fixed frames
+POLIASTRO_BODIES = [
+    Sun_poliastro,
+    Mercury_poliastro,
+    Venus_poliastro,
+    Earth_poliastro,
+    Moon_poliastro,
+    Mars_poliastro,
+    Jupiter_poliastro,
+    Saturn_poliastro,
+    Uranus_poliastro,
+    Neptune_poliastro,
+]
+POLIASTRO_ICRS_FRAMES = [
+    HCRS_poliastro,
+    MercuryICRS_poliastro,
+    VenusICRS_poliastro,
+    GCRS_poliastro,
+    MoonICRS_poliastro,
+    MarsICRS_poliastro,
+    JupiterICRS_poliastro,
+    SaturnICRS_poliastro,
+    UranusICRS_poliastro,
+    NeptuneICRS_poliastro,
+]
+POLIASTRO_FIXED_FRAMES = [
+    SunFixed_poliastro,
+    MercuryFixed_poliastro,
+    VenusFixed_poliastro,
+    ITRS_poliastro,
+    MoonFixed_poliastro,
+    MarsFixed_poliastro,
+    JupiterFixed_poliastro,
+    SaturnFixed_poliastro,
+    UranusFixed_poliastro,
+    NeptuneFixed_poliastro,
+]
+
+
+# Collect both API data in two dictionaries
+OREKIT_BODIES_AND_FRAMES = dict(
+    zip(BODIES_NAMES, zip(OREKIT_BODIES, OREKIT_FIXED_FRAMES))
+)
+POLIASTRO_BODIES_AND_FRAMES = dict(
+    zip(
+        BODIES_NAMES,
+        zip(POLIASTRO_BODIES, POLIASTRO_ICRS_FRAMES, POLIASTRO_FIXED_FRAMES),
+    )
+)
+
+
+# Macros for J2000 and ICRF frame from Orekit API
+J2000_OREKIT = AbsoluteDate.J2000_EPOCH
+ICRF_FRAME_OREKIT = FramesFactory.getICRF()
+GCRF_FRAME_OREKIT = FramesFactory.getGCRF()
+ITRF_FRAME_OREKIT = FramesFactory.getITRF(IERSConventions.IERS_2010, False)
+
+
+# Some of tests are marked as XFAIL since Orekit implements the data from IAU
+# WGCCRE 2009 report while poliastro uses IAU WGCCRE 2015 one
+
+
+@pytest.mark.parametrize("r_vec", R_SET)
+@pytest.mark.parametrize("v_vec", V_SET)
+@pytest.mark.parametrize(
+    "body_name",
+    [
+        "Sun",
+        pytest.param("Mercury", marks=pytest.mark.xfail),
+        "Venus",
+        "Moon",
+        pytest.param("Mars", marks=pytest.mark.xfail),
+        "Jupiter",
+        "Saturn",
+        "Uranus",
+        pytest.param("Neptune", marks=pytest.mark.xfail),
+    ],
+)
+def validate_from_body_intertial_to_body_fixed(body_name, r_vec, v_vec):
+
+    # poliastro: collect body information
+    (
+        BODY_POLIASTRO,
+        BODY_ICRF_FRAME_POLIASTRO,
+        BODY_FIXED_FRAME_POLIASTRO,
+    ) = POLIASTRO_BODIES_AND_FRAMES[body_name]
+
+    # Compute for the norm of position and velocity vectors
+    r_norm, v_norm = [norm(vec) for vec in [r_vec, v_vec]]
+    R = BODY_POLIASTRO.R.to(u.m).value
+
+    # Correction factor to normalize position and velocity vectors
+    k_r = R / r_norm if r_norm != 0 else 1.00
+    k_v = 1 / v_norm if v_norm != 0 else 1.00
+
+    # Make a position vector who's norm is equal to the body's radius. Make a
+    # unitary velocity vector. Units are in [m] and [m / s].
+    rx, ry, rz = [float(k_r * r_i) for r_i in r_vec]
+    vx, vy, vz = [float(k_v * v_i) for v_i in v_vec]
```

Comment:
Same as above

Reply:
In this case this is required as the orekit API is being used. Guess something related to the Java virtual machine or the wrapping nature of the package...